contrib/pzstd/utils/Portability.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under both the BSD-style license (found in the
+ * LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ * in the COPYING file in the root directory of this source tree).
+ */
+
+#pragma once
+
+#include <algorithm>
+
+// Required for windows, which defines min/max, but we want the std:: version.
+
+#ifdef min
+#undef min
```

Comment:
very minor : 
I believe you can `#undef` unconditionally, whether `min` is already defined or not.
This is explicitly allowed by the standard,
and I believe no compiler warning will be triggered.